Ninjathulhu/Ninjathulhu.Game/Entity.cs

```diff
@@ -3,80 +3,75 @@
 
 namespace Ninjathulhu.Game
 {
-    class EntityComponents
+    public class EntityComponents
     {
-        private Entity Entity;
+        private readonly Entity _entity;
 
-        private Dictionary<Type, Component> Components = new Dictionary<Type, Component>();
+        private readonly Dictionary<Type, Component> _components = new Dictionary<Type, Component>();
```

Comment:
`_privateMember` is a pretty common naming pattern in C#, I think. Hope you don't mind.